README.md

```diff
@@ -79,6 +79,23 @@ The returned `TaskResult` can be interrogated to query the current state of the
 
 If the task takes arguments, these can be passed as-is to `enqueue`.
 
+#### Transactions
+
+By default, it's up to the backend to determine whether the task should be enqueued immediately (after calling `.enqueue`) or wait until the end of the current database transaction (if there is one).
+
+This can be configured using the `ENQUEUE_ON_COMMIT` setting. `True` and `False` force the behaviour, and `None` is used to let the backend decide.
```

Comment:
Question: Should this be configurable?

The only use case I can think of for not waiting for a transaction is if if you want it to enqueue regardless of if later code raises an exception.

Reply:
Yes, but better to specify a default. `True` is probably the least surprising.